tests/unit/core/test_dispatch.py

```diff
@@ -44,3 +51,15 @@ def test_concat_columns(device):
     data_frames = [df1, df2]
     res = concat_columns(data_frames)
     assert res.columns.to_list() == ["a", "b", "c"]
+
+
+@pytest.mark.skipif(not cp, reason="Cupy not available")
+def test_pandas_cupy_combo():
```

Comment:
is there intended to be a call to the `make_df` function in this test?

Reply:
Originally the idea was to show that the code in make_df was correct. But I will add the make_df call here also. And compare both DFs.